### PR TITLE
ref: Delete assorted dead and orphaned code

### PR DESCRIPTION
**Pull request by tkaemming**
This removes a bunch of dead code from `sentry.utils.javascript` that I encountered when I was researching where we are going to need to support environment parameters for TSDB, and the code that was only used by code that was deleted (and so on.)

### CODE REVIEW

File: src/sentry/templates/sentry/groups/public_details.html [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-    {% handle_before_events request group %}
-
-    <div id="details">
-        <div id="event_list" class="inactive"></div>
-    </div>
-
-    <section class="body">
-        {% block main %}
-
-            {% with event|get_rendered_interfaces:request as interface_list %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This template tag didn't exist any longer

File: src/sentry/templates/sentry/groups/public_details.html [hunk 2]
```diff
@@ -1,65 +0,0 @@
-{% extends "sentry/groups/details.html" %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This file is still referenced, I didn't check to see if those references are valid

File: src/sentry/templates/sentry/groups/public_details.html [hunk 3]
```diff
@@ -1,65 +0,0 @@
...
-
-{% load i18n %}
-{% load sentry_activity %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This was the last use of this library and it is deleted as part of this patch

File: src/sentry/templates/sentry/groups/public_details.html [hunk 4]
```diff
@@ -1,65 +0,0 @@
...
-
-{% block page_header_block %}
-    <section id="page-header" class="toolbar" style="height:5px;">
-    </section>
-{% endblock %}
-
-{% block bodyclass %}{% endblock %}
-
-{% block main %}
-    {% handle_before_events request group %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This template tag was removed

File: src/sentry/templates/sentry/groups/public_details.html [hunk 5]
```diff
@@ -1,65 +0,0 @@
...
-                    <a class="btn pull-left prev {% if not next_event %} disabled{% endif %}"{% if next_event %} href="{% url 'sentry-group-event' group.organization.slug group.project.slug group.id next_event.id %}"{% endif %}><span></span> {% trans "Newer Event" %}</a>
-                    <a class="btn pull-right next {% if not prev_event %} disabled{% endif %}"{% if prev_event %} href="{% url 'sentry-group-event' group.organization.slug group.project.slug group.id prev_event.id %}"{% endif %}>{% trans "Older Event" %} <span></span></a>
-                    <h4>Event at {{ event.datetime }} {{ event.datetime.tzinfo }}</h4>
-                </div>
-
-                <div class="alert alert-block alert-info">
-                    <p>{% trans "You are viewing a publicly available version of this event's data. Some information may not be available." %}</p>
-                </div>
-
-                {% if event.has_two_part_message %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This method was deleted

File: src/sentry/templates/sentry/groups/public_details.html [hunk 6]
```diff
@@ -1,65 +0,0 @@
...
-                      <div class="box-header">
-                          <h3>{% trans "Full message" %}</h3>
-                      </div>
-                      <div class="box-content with-padding">
-                        <pre id="full-message">{{ event.message }}</pre>
-                      </div>
-                    </div>
-                {% endif %}
-
-                {% include "sentry/partial/_event_details.html" %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This partial did not exist

File: src/sentry/templates/sentry/groups/public_details.html [hunk 7]
```diff
@@ -1,65 +0,0 @@
...
-            {% endwith %}
-        {% endblock %}
-    </section>
-{% endblock %}
-
-{% block content_after %}
-    <script type="text/javascript">
-    $(document).ready(function() {
-        new app.GroupDetailsPage({
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This class did not exist

File: src/sentry/templates/sentry/groups/public_details.html [hunk 8]
```diff
@@ -1,65 +0,0 @@
...
-            group: {{ group|to_json:request|safe }},
```
Comment (tkaemming) [inline src/sentry/templates/sentry/groups/public_details.html]: This was the only `to_json` that took a object that would have hit the "transformer" code in `sentry.utils.javascript`

File: src/sentry/templates/sentry/users/details.html [hunk 9]
```diff
@@ -1,63 +0,0 @@
...
-            {% for key, value in tag.data.data.iteritems %}
-                <dt>{{ key|titlize }}</dt>
-                <dd>{{ value }}</dd>
-            {% endfor %}
-        </dl>
-    {% endif %}
-{% endblock %}
-
-{% block main %}
-    {% paginator event_list from request as event_list per_page EVENTS_PER_PAGE %}
```
Comment (tkaemming) [inline src/sentry/templates/sentry/users/details.html]: This was the last use of this `EVENTS_PER_PAGE` "constant"

**Comment by dcramer** [comment]
💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯

File: src/sentry/templates/sentry/users/details.html (hunk 9 above)
Comment (tkaemming) [inline src/sentry/templates/sentry/users/details.html]: Unsure where `paginator` is/was defined

**Review by dcramer — APPROVED** [review]
id make sure plugins and getsentry are green with this change

File: src/sentry/templates/sentry/users/details.html [hunk 10]
```diff
@@ -1,63 +0,0 @@
...
-        <div class="btn-group pull-right">
-            <a class="btn prev{% if not event_list.paginator.has_previous %} disabled{% else %}" href="?p={{ event_list.paginator.previous_page }}{% endif %}"><span>{% trans "Previous" %}</span></a>
-            <a class="btn next{% if not event_list.paginator.has_next %} disabled{% else %}" href="?p={{ event_list.paginator.next_page }}{% endif %}"><span>{% trans "Next" %}</span></a>
-        </div>
-    </div>
-
-    <div id="event_list"></div>
-
-    <script>
-    new app.GroupListView({
```
Comment (tkaemming) [inline src/sentry/templates/sentry/users/details.html]: This class did not exist

**Review by mattrobenolt — APPROVED** [review]
If tests pass, :cookie:.

If not, 🥉.

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request merged**